README.org

```diff
@@ -1,17 +1,46 @@
 #+TITLE: Ren'Py mode
 
-PyTom's old major mode for Ren'Py, previously maintained by treymerkley, now maintained by me.
+A major mode for editing Ren'Py files.
 
 ** Installation
 
   Either install within Emacs directly from MELPA or:
+
   * Download/clone this repo to your desired location.
+
   * In Emacs, use the command =M-x package-install-file= and point it to the renpy.el file location
-    * This will install it to your =~/.emacs.d/elpa/= directory (or your custom location if you've manually set a different =package-user-dir= in Emacs)
+
+    * This will install it to your =~/.emacs.d/elpa/= directory (or your custom location
+      if you've manually set a different =package-user-dir= in Emacs)
+
   * In your .emacs file, add =(require 'renpy)=
```

Comment:
I don't think `(require 'renpy) is actually required, because the autoloads should still be generated and processed when installing from a file.